plugins/candela/plugin_tests/candelaSpec.js

```diff
@@ -19,6 +19,10 @@ describe('Test the candela UI.', function () {
         });
         girderTest.waitForLoad();
```

Comment:
To be clear, it would look like this, correct?

```
        runs(function () {
            expect($('#g-user-action-menu.open').length).toBe(0);
            $('.g-user-text>a:first').click();
            expect($('#g-user-action-menu.open').length).toBe(1);
            $('a.g-my-folders').click();
        });
```